src/App.jsx

```diff
@@ -21,23 +22,38 @@ import User from "./pages/User.jsx";
 class App extends React.Component{
     constructor(props){
         super(props);
+
+        this.updateCurrentUser = this.updateCurrentUser.bind(this)
+        this.signOut = this.signOut.bind(this)
+    }
+
+    updateCurrentUser(userInfo) {
+        localStorage.setItem('currentUser', userInfo.name);
+    }
+    
+    signOut() {
+        this.setState({ currentUser: '' });
```

Comment:
Do we use currentUser?

Reply:
Sisi, lo agregué para mostrar el nombre del usuario logeado 
![imagen](https://user-images.githubusercontent.com/66975141/146223332-dc06c5a4-5eb7-40ca-8c3c-fa700920a48a.png)